metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/bolt/GenericEnrichmentBolt.java

```diff
@@ -161,6 +167,7 @@ protected void initializeStellar() {
     stellarContext = new Context.Builder()
                          .with(Context.Capabilities.ZOOKEEPER_CLIENT, () -> client)
                          .with(Context.Capabilities.GLOBAL_CONFIG, () -> getConfigurations().getGlobalConfig())
+                         .with(Context.Capabilities.GEO_IP, () -> GeoLiteDatabase.INSTANCE)
```

Comment:
Since the INSTANCE is static, is there a reason to not just refer to it directly in the Stellar function rather than adding a capability?  I always envisioned capabilities to be for configs which were changing at runtime.